packages/flame_texturepacker/lib/src/texture_packer_sprite.dart

```diff
@@ -68,4 +80,52 @@ class TexturePackerSprite extends Sprite {
 
   /// The [degrees] field (angle) represented as radians.
   double get angle => radians(degrees.toDouble());
+
+  late final Decorator _decorator;
+  final Transform2D _transform = Transform2D();
+
+  // Used to avoid the creation of new Vector2 objects in render.
+  static final _tmpRenderPosition = Vector2.zero();
+  static final _tmpRenderSize = Vector2.zero();
+
+  @override
+  void render(
```

Comment:
Sound all this be skipped if there is no rotation set?
So that `super.render` is called if there is no rotation, since that should be more performant?

Reply:
This is a good idea. I updated the code